xwiki-pro-macros-ui/src/main/resources/XWiki/Macros/Team.xml

```diff
@@ -394,7 +394,8 @@
       <cache>long</cache>
     </property>
     <property>
-      <code>.xwikiteam .xwikiteam-ul {
+      <code>
```

Comment:
This is an unnecessary change.
Note that it's important that the changes in a commit to be only about that issue. This way the commit will be clean, easy to read and you won't make the `blame` feature harder to follow. So you need to check at the end the files changes, in case you forgot something in there.